lib/pure/strformat.nim

```diff
@@ -296,6 +296,29 @@ to be injected into the context where ``myTemplate`` is expanded,
 everything is wrapped in a ``block``.
 
 
+Backslash support
+=================
+
+Nim ``strformat`` supports Backslash ``\``:
```

Comment:
No, it doesn't. That's a feature of the "triple string literal", not of ``strformat``.

Reply:
I know, the point is that it can be used, python one does not support that, it exits with error:

```python
>>> print( f"""{ "yep\nope" }""" )
Error: f-string expression part cannot include a backslash.
```

:)